app/assets/stylesheets/profiles.scss

```diff
@@ -0,0 +1,12 @@
+.user_avatar {

```

Comment:
actually .. something is wrong here ....
all styles are global and will impact all pages
put them in some parent class